developer-docs-site/docs/guides/handle-aptos-errors.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Handling Errors in Aptos and Move"
+slug: "handle-aptos-errors"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Handling Errors in Aptos and Move
+
+This page catalogs common errors encountered in the Aptos blockchain and explains how to resolve them wherever possible. As with all software, the code itself is the source of truth for error handling and will always contain entries not found here.
+
+Instead, this matrix aims to help you address those errors most typically found, misunderstood, or both.
```

Comment:
Suggest combining this "Instead, this ..." sentence with the above, since it's all one continuous idea. It feels like a broken twig now.

Reply:
Done. Thanks.